src/main/java/seedu/ddd/model/contact/common/Contact.java

```diff
@@ -111,7 +111,7 @@ public void removeEvent(Event event) {
     }
 
     /**
-     * Returns true if both persons have the same name.
+     * Returns true if both persons have the same name and phone number.
```

Comment:
Was this meant to be "or" instead of "and"?

Reply:
I think based on our previous conversation we decided "and".

Reply:
I think i implemented "or"